### DISCUSSION TITLE
How many countries are there in ⬇️🌎?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#798] [Galaxy Trivia] [answered]
*List them all as emojis*

**Comment by peterbaccaro** [comment]
There are **32 countries** in the southern hemisphere:

🇦🇴 Angola
🇦🇷 Argentina
🇦🇺  Australia
🇧🇴 Bolivia
🇧🇼 Botswana
🇧🇮 Burundi
🇨🇱 Chile
🇰🇲 Comoros
🇹🇱 East Timor
🇸🇿 Eswatini
🇫🇯 Fiji
🇱🇸 Lesotho
🇲🇬 Madagascar
🇲🇼 Malawi
🇲🇺 Mauritius
🇲🇿 Mozambique
🇳🇷 Nauru
🇳🇿 New Zealand
🇵🇬 Papua New Guinea
🇵🇾 Paraguay
🇵🇪 Peru
🇷🇼 Rwanda
🇼🇸 Samoa
🇸🇨 Seychelles
🇸🇧 Solomon Islands
🇿🇦 South Africa
🇹🇿 Tanzania
🇹🇻 Tuvalu
🇺🇾 Uruguay
🇻🇺 Vanuatu
🇿🇲 Zambia
🇿🇼 Zimbabwe

**Comment by SANKARAMDAS** [comment]
There are 32 countries in the Southern Hemisphere from five of the Earth's continents.


1 | Angola   🇦🇴
2 | Argentina    🇦🇷
3 | Australia   🇦🇺
4 | Bolivia   🇧🇴
5 | Botswana  🇧🇼
6 | Burundi   🇧🇮
7 | Chile   🇨🇱
8 | Comoros   🇰🇲
9 | East Timor  🇹🇱
10 | Eswatini  🇸🇿
11 | Fiji  🇫🇯
12 | Lesotho  🇱🇸
13 | Madagascar  🇲🇬
14 | Malawi   🇲🇼
15 | Mauritius   🇲🇺
16 | Mozambique   🇲🇿
17 | Nauru   🇳🇷
18 | New Zealand   🇳🇿
19 | Papua New Guinea   🇵🇬
20 | Paraguay  🇵🇾
21 | Peru   🇵🇪
22 | Rwanda  🇷🇼
23 | Samoa     🇼🇸
24 | Seychelles   🇸🇨
25 | Solomon Islands  🇸🇧
26 | South Africa   🇿🇦
27 | Tanzania   🇹🇿
28 | Tuvalu  🇹🇻
29 | Uruguay  🇺🇾
30 | Vanuatu vu
31 | Zambia   zm
32 | Zimbabwe  zw

**Comment by ParanoidUser** [comment]
Great job! And how many of them are actually in South America?

**Reply by peterbaccaro** [reply, marked as answer]
**6** of the countries in the southern hemisphere are in South America:

1. 🇦🇷 Argentina
2. 🇧🇴 Bolivia
3. 🇨🇱 Chile
4. 🇵🇾 Paraguay
5. 🇵🇪 Peru
6. 🇺🇾 Uruguay
